api/src/main/java/org/openmrs/module/providermanagement/ProviderManagementUtils.java

```diff
@@ -99,6 +104,66 @@ public static Date clearTimeComponent(Date date) {
 
         return cal.getTime();
     }
+
+    public static List<ProviderPersonRelationship> getSupervisors(Provider provider)
```

Comment:
There are several gerSupervisersForProvider methods in the ProviderManagementService... looks like these two methods run on top of that?  Can you provide a few lines of documentation of why these are needed and what additional funcitonality they provider?